packages/blade/src/components/BaseHeaderFooter/BaseHeader.tsx

```diff
@@ -155,15 +175,17 @@ const _BaseHeader = ({
         touchAction="none"
         {...webOnlyEventHandlers}
       >
-        <BaseBox display="flex" flexDirection="row" alignItems="center" userSelect="none">
+        <BaseBox display="flex" flexDirection="row" userSelect="none">
```

Comment:
This change looks okay on mobile bottomsheet?

Reply:
Lots of alignment changes in header, hope you've checked for both web & mobile on bottomsheet, dropdown & modal

Reply:
Yup seems to look good

<img width="377" alt="image" src="https://github.com/razorpay/blade/assets/30949385/461b4f62-6cbd-45c3-aeb7-5517c076b57f">